core/new-gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -41,19 +38,26 @@ export class WorkspaceComponent implements AfterViewInit, OnInit, OnDestroy {
   public showResultPanel: boolean = false;
   userSystemEnabled = environment.userSystemEnabled;
 
+  screenWidth = window.innerWidth;
+  width = 200;
+  id = -1;
+  disabled = false;
+
+  onResize({ width }: NzResizeEvent): void {
+    cancelAnimationFrame(this.id);
+    this.id = requestAnimationFrame(() => {
+      this.width = width!;
+    });
+  }
+
   constructor(
     private userService: UserService,
     private resultPanelToggleService: ResultPanelToggleService,
-    // list additional services in constructor so they are initialized even if no one use them directly
-    private schemaPropagationService: SchemaPropagationService,
-    private autoAttributeCorrectionService: AutoAttributeCorrectionService,
```

Comment:
why are we removing those services? according the the comment, it is better to keep them?

Reply:
Added. Removed by IDE.

Reply:
Added.